--- a/builtin/init-db.c	2020-06-12 21:28:20.760631039 +0100
+++ b/builtin/init-db.c	2020-06-12 21:28:35.860791116 +0100
@@ -233,14 +233,14 @@
 		die("failed to set up refs db: %s", err.buf);

 	/*
-	 * Create the default symlink from ".git/HEAD" to the "master"
+	 * Create the default symlink from ".git/HEAD" to the "production"
 	 * branch, if it does not exist yet.
 	 */
 	path = git_path_buf(&buf, "HEAD");
 	reinit = (!access(path, R_OK)
 		  || readlink(path, junk, sizeof(junk)-1) != -1);
 	if (!reinit) {
-		if (create_symref("HEAD", "refs/heads/master", NULL) < 0)
+		if (create_symref("HEAD", "refs/heads/production", NULL) < 0)
 			exit(1);
 	}

